app/helpers/token.js

```diff
@@ -1,7 +1,18 @@
 const jwt = require('jsonwebtoken');
 const { secret } = require('../../config/constants');
+const logger = require('../logger/index');
 
-const signToken = email => jwt.sign({ email }, secret);
-const verifyToken = token => jwt.verify(token, secret);
+const { EXPIRATION_TIME } = process.env;
```

Comment:
It'd be better to call this `JWT_EXPIRATION_TIME ` instead. Also put this in `config/environment`.
Normally env variables are all taken from `process.env` in that file.